src/Fetch.js

```diff
@@ -342,7 +338,11 @@ function fetchXHR(fetch, onsuccess, onerror, onprogress, onreadystatechange) {
   }
 
   xhr.onload = (e) => {
-    saveResponse(fetchAttrLoadToMemory && !fetchAttrStreamData);
+    // check if xhr was aborted by user and don't try to call back
+    if (!(id in Fetch.xhrs)) { 
+      return;
+    }
+    saveResponse(fetchAttrLoadToMemory && HEAPU32[fetch + {{{ C_STRUCTS.emscripten_fetch_t.data }}} >> 2] === 0);
```

Comment:
Why stop using `fetchAttrStreamData` on this line?

Reply:
Because fetchAttrStreamData is not working like expected, it does not save the data, so we just check data pointer - if it is null, then we should save it here.